src/main/java/core/basesyntax/models/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.models;
+
+import core.basesyntax.Figure;
+
+public class Circle implements Figure {
+    private final String color;
+    private final double radius;
+
+    public Circle(String color, double radius) {
+        this.color = color;
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Color: " + color
+                + "Type: Circle "
+                + "Radius: " + radius);

```

Comment:
The 'draw()' method should have all behavior related to drawing the figure. It seems like the 'Type: Circle ' string is missing a line break or separator before it, which could lead to concatenated output like 'Color: redType: Circle'. Consider adding a comma, a line break, or some other separator to ensure the output is clearly readable and formatted correctly.